resorts/views/resort_list.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+#General imports
+from collections import defaultdict
+from django.core.exceptions import ObjectDoesNotExist
+
+#Import Resort and ConditionsReport models
+from ..models import Resort, Conditions
+
+#Imports for using Google Maps
+import googlemaps
+import json
+
+#Google Maps requires a client key to work
+GMAPS = googlemaps.Client(key='AIzaSyBRrCgnGFkdRY-Z1hX6xaxoUFBczNI2664')
+
+
+def get_resort_list(user_address, selected_resorts, number_to_display=5, order_on='snow_in_past_24h'):
+#Given a user address and selected resorts, return an ordered list of resorts for display
```

Comment:
Please comment functions with docstring (aka triple quotes """). See http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for examples. You should include a description, list of paramters (Args: ), and return value (Returns: ). Thank you 😺